.github/workflows/publish.yml

```diff
@@ -40,13 +46,13 @@ jobs:
       - name: Run Build
         run: task
   gh-release:
-    if: github.event_name != 'pull_request'
+    if: github.event_name == 'push'
```

Comment:
isn't push event even when someone pushes on pull requests?

Reply:
Since we filter the push event to the `master` branch, not only does [push not run on PRs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push), but it should also only run in the master branch.